lib/compress/zstdmt_compress.c

```diff
@@ -1818,6 +1835,60 @@ static int ZSTDMT_tryGetInputRange(ZSTDMT_CCtx* mtctx)
     return 1;
 }
 
+/**
+ * Searches through the input for a synchronization point. If one is found, we
+ * will instruct the caller to flush, and return the number of bytes to load.
+ * Otherwise, we will load as many bytes as possible and continue as normal.
+ */
+static size_t findSynchronizationPoint(ZSTDMT_CCtx* mtctx, ZSTD_inBuffer* input, ZSTD_EndDirective* endOp) {
```

Comment:
I see multiple exit points in this function.
Should `*endOp` be initialized to `ZSTD_e_continue` ?
Or is it presumed already initialized by the caller ? In which case, should it be asserted ?